sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -36,4 +43,38 @@ export class InstitutionLocationAESTController extends BaseController {
       institutionId,
     );
   }
+
+  /**
+   * Controller method to retrieve institution location by id.
+   * @param institutionId
+   * @param locationId
+   * @returns institution location.
+   */
+  @Get(":institutionId/:locationId")
+  async getInstitutionLocation(
+    @Param("institutionId") institutionId: number,
+    @Param("locationId") locationId: number,
+  ): Promise<InstitutionLocationFormAPIOutDTO> {
+    return this.locationControllerService.getInstitutionLocation(
+      institutionId,
+      locationId,
+    );
+  }
+
+  /**
+   * Update an institution location.
+   * @param locationId
+   * @param payload
+   * @returns number of updated rows.
+   */
+  @ApiBadRequestResponse({
+    description: "Invalid request to update the institution location.",
```

Comment:
is this payload bad exception?

Reply:
are we adding payload bad exception API tags in other controllers? @andrewsignori-aot  @dheepak-aot  @henryoforeh-aot  @guru-aot